neo/Storage/Interface/DBInterface.py

```diff
@@ -16,13 +40,15 @@ class DBCollection:
     Changed = []
     Deleted = []
```

Comment:
I realise this is old code that already had these class variables. I know this will sound contradicting to what I just said above, but leave them here for now as I have a commit in another branch that should remove all of these and make them instance variables

Reply:
Not touching those.